### PR TITLE
1.0: use core plugin for roundrobin example

### PR DESCRIPTION
**Pull request by kenhys**
Using non standard output plugin makes a new user confused.

Closes: #325

### CODE REVIEW

File: output/roundrobin.md [hunk 1]
```diff
@@ -11,20 +11,32 @@ It is included in Fluentd's core.
   @type roundrobin
 
   <store>
-    @type tcp
-    host 192.168.1.21
+    @type http
+    endpoint http://192.168.1.21
+    <format>
```
Comment (ashie) [inline output/roundrobin.md:16]: `format` and ` json_array` aren't needed, because they aren't mandatory options.
Since this page is a document for roundrobin plugin, information that is not related to it should be suppressed as much as possible.

Comment (kenhys) [reply output/roundrobin.md]: It should be, thanks!

**Comment by kenhys** [comment]
Removed not mandatory options.

**Review by ashie — APPROVED** [review]
LGTM
